features/admin.webhooks.v1/api/update-webhook-metadata.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AsgardeoSPAClient, HttpClientInstance } from "@asgardeo/auth-react";
+import { RequestConfigInterface } from "@wso2is/admin.core.v1/hooks/use-request";
+import { store } from "@wso2is/admin.core.v1/store";
+import { IdentityAppsApiException } from "@wso2is/core/exceptions";
+import { HttpMethods } from "@wso2is/core/models";
+import { AxiosError, AxiosResponse } from "axios";
+import { WebhooksConstants } from "../constants/webhooks-constants";
+import {
+    WebhookMetadataUpdateRequestInterface,
+    WebhookMetadataUpdateResponseInterface
+} from "../models/webhook-settings";
+
+const httpClient: HttpClientInstance = AsgardeoSPAClient.getInstance().httpRequest.bind(
+    AsgardeoSPAClient.getInstance()
+);
+
+/**
+ * Update webhook metadata.
+ * @param webhookMetadata - Webhook metadata update request payload.
+ * @returns Promise containing the response.
+ * @throws Throws an IdentityAppsApiException if the request fails.
+ */
+const updateWebhookMetadata = (
+    webhookMetadata: WebhookMetadataUpdateRequestInterface
+): Promise<WebhookMetadataUpdateResponseInterface> => {
+    const requestConfig: RequestConfigInterface = {
+        data: webhookMetadata,
+        headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.PATCH,
+        url: `${store.getState().config.endpoints.webhooksMetadata}`
```

Comment:
no need to use template literals here.

```suggestion
        url: store.getState().config.endpoints.webhooksMetadata
```

Reply:
This is updated with d4ba4ce5f28de4329e3157e0b963ac69abe7e38e